### PR TITLE
Heal-38-Doctors should be able to manage the appointments   UI

### PR DESCRIPTION
**Pull request by Shruthi-2306**
Screen Shot 2023-03-18 at 2 18 07 PM

### CODE REVIEW

File: config/constants.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const healthBlockAddress=process.env.HEALTHBLOCK_ADDRESS || "0x6Fd2B45F4Eff401654028c5097fdE9c6a8f07298";
+const healthBlockAddress=process.env.HEALTHBLOCK_ADDRESS || "0xf701f7f0f2a1536580D2bAB16C25b29CB2874641";
```
Comment (varunalla) [inline config/constants.js]: Nit! 
please use environmental variables to build.

File: client/src/components/Doctor/DoctorDashboard.tsx [hunk 2]
```diff
@@ -9,8 +9,15 @@ const DoctorDashboard: FunctionComponent<{}> = () => {
...
+    }
     return (
+    <div>
+        <button className="inline-flex items-center px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800" onClick={() => appointmentHandler()}>Appointments</button>
+        
         <div className="flex flex-row min-h-screen justify-center items-center">
+
             <div className="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow-md dark:bg-gray-800 dark:border-gray-700">
                 <div className="flex flex-col items-center pb-10">
                     {/* <img className="w-24 h-24 mb-3 rounded-full shadow-lg" src="/docs/images/people/profile-picture-3.jpg" alt="Patient image" />*/}
```
Comment (varunalla) [inline client/src/components/Doctor/DoctorDashboard.tsx]: remove comment if possible

File: client/src/components/Doctor/DoctorAppointments.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    },
+    {
+      name: "Hasini",
+      email: "hasini@gmail.com",
+      date: "03/27/2023",
+      time: "10.00 AM",
+      status: "Confirmed",
+      id: "1",
+    },
+  ];
```
Comment (varunalla) [inline client/src/components/Doctor/DoctorAppointments.tsx:23]: i hope this will be integrated with backend!

Comment (Shruthi23) [reply client/src/components/Doctor/DoctorAppointments.tsx:23]: Yes. This PR is only for UI development. I will have the API integration and the functionality addressed in another ticket.

**Review by varunalla — APPROVED** [review]
approved! make sure the back end integration pr is up by tomorrow!

**Pull request merged**